templates/setup.rb

```diff
@@ -0,0 +1,5 @@
+setup :secret_key_base
+env 'HOST', provide('Host', default: 'localhost:3000')
+env 'ASSET_HOST', provide('Asset host', default: 'localhost:3000')
+
+rake 'db:setup'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.